### PR TITLE
Update styling

### PR DESCRIPTION
**Pull request by rmparr**
I have added new colors and fonts per the design.

The footer has been redesigned, and the new fonts have been added throughout the page.

Merging this **will** require some work in Alchemy Admin to create the new elements.

Footer-desktop
Footer-mobile-landscape
Footer-mobile-portrait

### CODE REVIEW

File: app/assets/stylesheets/bootstrap_custom.scss [hunk 1]
```diff
@@ -46,8 +46,8 @@ $link-hover-decoration: underline !default;
 //
 //## Font, line-height, and color for body text, headings, and more.
 
-$font-family-sans-serif:  "Helvetica Neue", Helvetica, Arial, sans-serif !default;
-$font-family-serif:       Georgia, "Times New Roman", Times, serif !default;
+$font-family-sans-serif:  Roboto;
```
Comment (mamhoff) [inline app/assets/stylesheets/bootstrap_custom.scss]: You want to provide a fallback here pretty please

File: app/views/alchemy/elements/_legal_editor.html.erb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<%= element_editor_for(element) do |el| %>
```
Comment (mamhoff) [inline app/views/alchemy/elements/_legal_editor.html.erb:1]: This should not be an Element IMO but rather a menu. I'll be making some menus this afternoon for Fundfinder; they're a new Alchemy Feature.

**Review by mamhoff — COMMENTED** [review]
Good work! I have one nit regarding using elements for menus :)

**Comment by mamhoff** [comment]
There's also an underline between the two social icons that probably should not be there.

File: app/views/alchemy/elements/_legal_editor.html.erb (hunk 2 above)
Comment (rmparr) [reply app/views/alchemy/elements/_legal_editor.html.erb:1]: PtW is still on Alchemy 4.1, I think menus were implemented in 4.4.

Upgrading Alchemy should definitely be on the roadmap.

Comment (mamhoff) [reply app/views/alchemy/elements/_legal_editor.html.erb:1]: Oh yeah

File: app/assets/stylesheets/bootstrap_custom.scss (hunk 1 above)
Comment (rmparr) [reply app/assets/stylesheets/bootstrap_custom.scss]: I have re-added the sans-serif fonts as the fallback.

**Comment by rmparr** [comment]
Done!

**Review by mamhoff — APPROVED** [review]
Cool, let's merge then

**Pull request merged**